split.py

```diff
@@ -392,7 +393,7 @@ def main(config_path, base_dir, target_path, modes, verbose, use_cache=True):
         to_write = [
             s
             for s in symbols.all_symbols
-            if s.referenced and not s.defined and not s.dead and not s.type == "func"
```

Comment:
Why this change? We shouldn't need to write out any symbols that aren't referenced